app/routes/dashboard.tsx

```diff
@@ -31,6 +32,7 @@ export const loader: LoaderFunction = async ({ request }) => {
 
 const navigation = [
   { name: 'Home', href: '/dashboard/', icon: HomeIcon },
+  { name: 'Profile', href: '/dashboard/profile', icon: UserIcon },
```

Comment:
You don't need this change. We already have a `View profile` button below in the navigation.
```suggestion
```

<img width="256" alt="Screenshot 2021-12-30 at 12 15 38" src="https://user-images.githubusercontent.com/6315466/147721316-d61a5b52-761c-48dd-ad8c-1fd35c8b4c0c.png">